### PR TITLE
auth_ldap : migration to the new API  

### PR DESCRIPTION
**Pull request by rga-odoo**
task: https://www.odoo.com/web#id=12141&view_type=form&model=project.task&action=333&active_id=131
pad : https://pad.odoo.com/p/r.xFYX5Mg1OvARmHLX

### CODE REVIEW

File: addons/auth_ldap/models/users_ldap.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+            return False
+
+        entry = False
+        filter = filter_format(self.ldap_filter, (login,))
+        try:
+            results = self.query(filter)
+
+            # Get rid of (None, attrs) for searchResultReference replies
+            results = [i for i in results if i[0]]
+            if results and len(results) == 1:
```
Comment (rga-odoo) [inline addons/auth_ldap/models/users_ldap.py]: if len(results) == 1:

File: addons/auth_ldap/models/users_ldap.py [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        :return: res_users id
+        :rtype: int
+        """
+
+        user_id = False
+        login = tools.ustr(login.lower().strip())
+        self.env.cr.execute("SELECT id, active FROM res_users WHERE lower(login)=%s", (login,))
+        res = self.env.cr.fetchone()
+        if res:
+            if res[1]:
```
Comment (rga-odoo) [inline addons/auth_ldap/models/users_ldap.py]: res and res[1]:

File: addons/auth_ldap/models/users_ldap.py (hunk 1 above)
Comment (dja-odoo) [inline addons/auth_ldap/models/users_ldap.py]: Done.

File: addons/auth_ldap/models/users_ldap.py (hunk 2 above)
Comment (dja-odoo) [inline addons/auth_ldap/models/users_ldap.py]: Done.

**Pull request closed without merging**